src/niweb/niweb/settings/common.py

```diff
@@ -188,7 +188,7 @@
 
 ########## MIDDLEWARE CONFIGURATION
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#middleware-classes
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = (
```

Comment:
Need to look into if this breaks the specific dev, prod settings or not.

Reply:
Seems to be fine and our plugins seems to check for both MIDDLEWARE_CLASSES and MIDDLEWARE.